ansible/vars/icds/icds_public.yml

```diff
@@ -36,7 +36,7 @@ active_sites:
   riakcs: True
   cas_ssl: True
 
-riak_ring_size: 128
+riak_ring_size: 64
```

Comment:
Thanks for the detailed explanation, @calellowitz. That's both interesting and disappointing. We should change this when we setup the new higher-capacity riak cluster for ICDS.